### ISSUE TITLE
Horizontal scrolling in rendered code blocks

### ISSUE THREAD

**Issue by thought2** [gcanti/docs-ts#33]
Currently `docs-ts` [uses prettier to format](https://github.com/gcanti/docs-ts/blob/3a98908f6fd9aa2d47fe23df57bf9422b7f346d1/src/Markdown.ts#L568) markdown before writing out the docs. 

The used options are:

```ts
const prettierOptions: prettier.Options = {
  parser: 'markdown',
  semi: false,
  singleQuote: true,
  printWidth: 120
}
```

The code blocks in Jekyll seem to work without horizontal scrolling only if lines don't exceed a 80 characters limit.

*Is this intended behavior?*

If not, I'd suggest one of those options:

- Configure Jekyll to use 120 characters as limit (don't know if this is possible)
- Configure Prettier to use 80 character limit

Or, and this Is what I would suggest:

- Don't format at all inside `docs-ts` and encourage the user to use [prettier-plugin-jsdoc](https://github.com/hosseinmd/prettier-plugin-jsdoc#readme). This is a much better editing experience. You have more control about the format, plus in the end the actual source file has the same formatting as the docs.

What do you think?

**Comment by IMax153** [comment]
This is a great point - I would favor changing the `prettier` config in the codebase rather than forcing users to install another dependency. @gillchristian - thoughts?

**Comment by gillchristian** [comment]
Yeah, I think 80 is quite the default for columns :ok_hand:

**Comment by thought2** [comment]
True, maybe it's the best to just change that value for the docs generation as @IMax153 suggests. That makes sure the code looks good in Jekyll. Plus it leaves the user the choice which `printWidth` to use in their codebase.

**Comment by waynevanson** [comment]
I assume that the prettier config in the code base is set to above 80 so that medium length type signatures don't look like a right mess.
